test/src/e2e_vm_tests/test_programs/should_fail/predicate_invalid_opcodes/Forc.toml

```diff
@@ -0,0 +1,8 @@
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+license = "Apache-2.0"
+name = "predicate_invalid_opcodes"
+entry = "main.sw"
+
+[dependencies]
+core = { path = "../../../../../../sway-lib-core" }
```

Comment:
Not really needed. Instead, add `implicit-std = false` under `[project]` above (and then rerun `forc fmt`)

Reply:
With `implicit-std = false`, I get the following error:
![predicate_invalid_opcodes](https://user-images.githubusercontent.com/1188095/200691115-80f6ee15-6a91-4c9c-91ef-38d26430f7c8.png)
The while loop at the end of the test is using operators from core::ops, so it looks like at least core is necessary.

Reply:
🤦‍♂️ sorry you're right!